CSCupRuntime/CSCupRuntime.csproj

```diff
@@ -17,21 +16,27 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <OutputPath>..\bin\Debug\</OutputPath>
+    <LangVersion>7.3</LangVersion>
```

Comment:
I I think you can move LangVersion to file ..\TypeCobol\Common.props
This will avoid to change all csproj files.

Reply:
Maybe not all csproj use the common.props file

Reply:
Right, I forgot about the common.props file. All done